test/e2e/date-select.test.js

```diff
@@ -5,45 +5,55 @@ import parsers from '../../common/parsers'
 import { fillInForm } from './_helpers'
 import { pmuUser } from './_roles'
 import {
-  home,
   incomingMoves,
   outgoingMoves,
   allocations,
   singleRequests,
 } from './_routes'
 import { page, movesDashboardPage } from './pages'
 
-const jumpToDate = async (t, paginatedPage, value) => {
-  const expectedDate = dateFns.format(parsers.date(value), 'yyyy-MM-dd')
-
-  await t.navigateTo(outgoingMoves)
-  await t.click(movesDashboardPage.nodes.pagination.dateSelectLink)
-  await fillInForm([
-    {
-      selector: page.nodes.dateSelectInput,
-      value,
-    },
-  ])
-  await page.submitForm()
-  await t.expect(page.getCurrentUrl()).contains(expectedDate)
-}
-
-fixture('Jump to date').beforeEach(async t => {
-  await t.useRole(pmuUser).navigateTo(home)
```

Comment:
There's no explicit testing of jump-to-date on the pmu screen, so this navigation should be removed :+1: